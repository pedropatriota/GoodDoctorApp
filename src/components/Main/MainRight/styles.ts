import styled from "styled-components"

export const Container = styled.div`
  width: 50%;
  height: 100vh;
  background-color: #370951;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  color: #000;
`
