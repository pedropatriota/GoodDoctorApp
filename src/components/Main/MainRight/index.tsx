import * as S from "./styles"
import SignUp from "../../SignUp"
import React from "react"

const MainRight = () => (
  <S.Container>
   <SignUp />
  </S.Container>
)

export default MainRight
