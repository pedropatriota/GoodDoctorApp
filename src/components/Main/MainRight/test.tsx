import { render, screen } from "@testing-library/react"
import MainRight from "./index"

describe("testing component MainRight", () => {
  it("Should render the heading", () => {
    const { container } = render(<MainRight />)

    expect(
      screen.getByRole("heading", { name: /Right Side/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })

  it("should render corretly the color", () => {
    const { container } = render(<MainRight />)

    expect(container.firstChild).toHaveStyle({ "background-color": "#bae1ff" })
  })
})
