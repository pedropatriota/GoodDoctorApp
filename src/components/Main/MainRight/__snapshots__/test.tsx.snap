// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing component MainRight Should render the heading 1`] = `
<div
  class="styles__Container-hXcVkP eavrgH"
>
  <h1>
    Right Side
  </h1>
</div>
`;
